.github/workflows/continuous-integration.yml

```diff
@@ -23,14 +22,15 @@ jobs:
           - "7.3"
           - "7.4"
           - "8.0"
+        dbal-version:
+          - "^2.12"
+          - "^3.0"
         dependencies:
           - "highest"
+          - "lowest"
```

Comment:
```suggestion
```

At least in `DoctrineModule`, we do only test lowest dependencies for the lowest PHP version, but not for all PHP versions. I think this would make sense here as well, as otherweise we have about 30 checks to run.

Reply:
Ok, I can do it, but I would like to understand what is wrong with 30 checks? It's add little more time to wait but also proves that works in many strange version combination. And it works now.

Reply:
@greg0ire I think we discussed this earlier and you wanted me to test lowest dependencies only with the lowest supported PHP version. What do you think of this, should we test lowest dependencies with all supported PHP versions? As we support both dbal 2 and 3, we have all tests running for both dbal 2 and 3. This results in 30 checks alltogether.

Reply:
Yes, I think one `lowest` build should be enough to figure out if you need to bump version constraints: if you need a specific commit, then you can figure out since which version it's available on all major versions. There are infinite version combinations, let's only test the cases that will catch the most bugs.

Reply:
Now CI config file is simple and any change in matrix will be correctly respect.
Removed version of `include` contains one position but missing `optional_dependencies`, because someone miss it. Too much work for us to maintain this IMO in future.

However I can change this if you still think to remove this.